runbotics/runbotics-scheduler/src/migrations/1732542379024-SequencesBump.ts

```diff
@@ -0,0 +1,99 @@
+import { GlobalVariable } from '#/scheduler-database/global-variable/global-variable.entity';
+import { ProcessEntity } from '#/scheduler-database/process/process.entity';
+import { Tag } from '#/scheduler-database/tags/tag.entity';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+interface CountResult { count: string; }
+type MaxResult = { max: string; } | null;
+
+export class SequencesBump1732542379024 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // Process sequence --------------
+        await queryRunner.query(`
+            SELECT COUNT(*) as count
+            FROM pg_class
+            WHERE relname = 'process_id_seq';
+        `).then((data: CountResult[]) => {
```

Comment:
why with native query instead of manager?

Reply:
I thought that those are postgress tables so pure SQL